src/framework/mpe/tests/multinotearticulationstest.cpp

```diff
@@ -30,7 +30,7 @@ using namespace mu;
 using namespace mu::mpe;
 using namespace mu::mpe::tests;
 
-class Engraving_MultiNoteArticulationsTest : public ::testing::Test
+class MultiNoteArticulationsTest : public ::testing::Test
```

Comment:
I believe Igor named them that way for nicer sorting in Qt Creator

Reply:
Thanks, I didn't realize that was intentional